### DISCUSSION TITLE
Is it possible to have my mention of an issue removed from that issue's activity?

### DISCUSSION THREAD

**Discussion by NoahStolk** [community/community#38715] [New to GitHub]
I just added an issue from a different repository of a package that I am using to the description of my PR, so I can keep track of it (I often leave to-do lists in my PR descriptions). However I noticed that my PR was instantly added to the activity list of that issue.

I don't want my silly hobby project to clog up the activity of that issue and distract the people who are working on it. I deleted the item from my to-do list, but the mention is still there. Is there a way to remove it?

**Comment by ossanna16** [comment]
Hi there @NoahStolk and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
